<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Add_course_menu</class>
 <widget class="QDialog" name="Add_course_menu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="10" column="0" colspan="3">
    <widget class="QPushButton" name="btn_cancel">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="lbl_hall">
     <property name="text">
      <string>Lecture Hall</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lbl_id">
     <property name="text">
      <string>Course ID</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QLineEdit" name="txt_hall"/>
   </item>
   <item row="7" column="2">
    <widget class="QLineEdit" name="txt_time"/>
   </item>
   <item row="0" column="2">
    <widget class="QLineEdit" name="txt_title"/>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="lbl_title">
     <property name="text">
      <string>Course Title</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLineEdit" name="txt_id"/>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="lbl_time">
     <property name="text">
      <string>time</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0" colspan="3">
    <widget class="QPushButton" name="btn_add">
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="lbl_prof">
     <property name="text">
      <string>Professor</string>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="QComboBox" name="combo_prof"/>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>txt_title</tabstop>
  <tabstop>txt_id</tabstop>
  <tabstop>txt_hall</tabstop>
  <tabstop>txt_time</tabstop>
  <tabstop>btn_add</tabstop>
  <tabstop>btn_cancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
